README.Rmd

```diff
@@ -49,51 +96,48 @@ devtools::install_github("cynkra/fledge")
 If you are used to making workflow packages (e.g. [devtools](https://usethis.r-lib.org/articles/articles/usethis-setup.html#use-usethis-or-devtools-in-interactive-work)) available for all your interactive work, 
 you might enjoy loading fledge in your [.Rprofile](https://rstats.wtf/r-startup.html#rprofile).
 
+### Once per package
+
+* Your package needs to have a remote that indicates the default branch (e.g. GitHub remote) _or_ to be using the same default branch name as your global/project `init.defaultbranch`.
+
+* If your package...
+
+   * is brand-new, remember to run `fledge::bump_version()` regularly.
+   * has already undergone some development, not too late to jump on the train! Run `fledge::bump_version()` and then `fledge::tag_version()`.
```

Comment:
mmh actually does one really need `fledge::tag_version()` at this stage? actually no since it's called by `bump_version()` via `finalize_version()`? @krlmlr

Reply:
`tag_version()` is called by `finalize_version()` . Can we showcase "jumping on the train" in the demo vignette?